### PR TITLE
Feature js prototypical cleanup

### PR DESCRIPTION
**Pull request by RachelRVasquez**
cleaned up JS to match app.js/search.js format

### CODE REVIEW

File: src/assets/fabricator/scripts/helpers.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	/**
+	 * Helper to cehck for className on an element
+	 */
+	hasClass: function( el, className ) {
+		//Check if element is undefined or null first
+		if ( 'undefined' === typeof el || null === el ) {
+			return false;
+		}
+
+		return el.className.indexOf( ' ' + className ) > -1;
```
Comment (fuhton) [inline src/assets/fabricator/scripts/helpers.js]: Why the ' ' ?

File: src/assets/fabricator/scripts/fabricator.js [hunk 2]
```diff
@@ -1,320 +1,216 @@
...
+		return this;
+	},
 
+	/**
+	 * Add fixed class to sidebar on scroll
+	 * @return {Object}
+	 */
+	fixSidebar: function() {
+		var dsHeaderTop  = document.querySelector( '.f-header-top' ),
+			dsHeader  = document.querySelector( '.f-header' ),
```
Comment (fuhton) [inline src/assets/fabricator/scripts/fabricator.js]: Spacing is weird.

File: src/assets/fabricator/scripts/app.js [hunk 3]
```diff
@@ -2,10 +2,16 @@
 
 'use strict';
 
-var search = require( './search' );
+var search = require( './search' ),
+	fabricator = require( './fabricator' );
```
Comment (fuhton) [inline src/assets/fabricator/scripts/app.js]: Spacing is weird.

Comment (RachelRVasquez) [inline src/assets/fabricator/scripts/app.js]: I'm checking the file in Sublime and the tabbing is correct - it's not a mixture of white space and tabs. Not sure why Github interprets the tabs this way. Looking into it.

File: src/assets/fabricator/scripts/fabricator.js (hunk 2 above)
Comment (RachelRVasquez) [inline src/assets/fabricator/scripts/fabricator.js]: Spacing should be fixed now with the .editconfig file.

File: src/assets/fabricator/scripts/app.js (hunk 3 above)
Comment (RachelRVasquez) [inline src/assets/fabricator/scripts/app.js]: Should be fixed now with the .editconfig file setting the tab size.

**Pull request merged**